asg-elb-service/main.tf

```diff
@@ -11,7 +11,10 @@
 terraform {
   # Live modules pin exact Terraform version; generic modules let consumers pin the version.
   # The latest version of Terragrunt (v0.25.1 and above) recommends Terraform 0.13.3 or above.
```

Comment:
does this comment still make sense given the new comment added by the tf14.rb script?

Reply:
I wonder why this used to pin to the exact version `0.13.3`, seems a bit weird. 

In any case, if we change the required_version, the old comment should definitely be removed.

Reply:
Ah, good catch. So, in `infra-modules` repos, we do recommend pinning an _exact_ Terraform version. That way, all the infra you deploy to real environments is on the same version, and no one accidentally updates to a newer version. E.g., If even one team member bumped from, say, `0.13.3` to `0.13.4`, and ran `apply`, then the state file would no longer work with older versions, and _everyone_ on the team and all the CI servers would have to upgrade. So I'll pin this to an exact 0.14.x version.

Reply:
PR updated. Please take a look!